packages/trader/src/App/Components/Elements/ContractAudit/contract-audit-item.jsx

```diff
@@ -1,21 +1,32 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { formatDate, formatTime } from '@deriv/shared';
+import { Text } from '@deriv/components';
 
 const ContractAuditItem = ({ icon, id, label, timestamp, value, value2 }) => (
     <div id={id} className='contract-audit__grid'>
         {icon && <div className='contract-audit__icon'>{icon}</div>}
         <div className='contract-audit__item'>
-            <span className='contract-audit__label'>{label}</span>
+            <Text className='contract-audit__label'>{label}</Text>
             <div className='contract-audit__value-wrapper'>
-                <span className='contract-audit__value'>{value}</span>
-                {value2 && <span className='contract-audit__value2'>{value2}</span>}
+                <Text weight='bold' size='xxs' line_height='m' color='prominent' className='contract-audit__value'>
+                    {value}
+                </Text>
+                {value2 && (
+                    <Text weight='bold' size='xxs' line_height='m' color='prominent' className='contract-audit__value2'>
```

Comment:
can we automate line height, based on the size?

Reply:
it is very different in every style. I think it is hard to make it automate.